flatflow/nemo/collections/nlp/data/language_modeling/megatron/megatron_batch_samplers.py

```diff
@@ -160,20 +157,22 @@ def set_epoch(self, epoch: int) -> None:
     def __iter__(self):
         indices = []
         model_group = parallel_state.get_model_parallel_group()
-        model_src_rank = flatflow.megatron.core.parallel_state.get_model_parallel_src_rank()
+        model_src_rank = torch.distributed.get_process_group_ranks(model_group)[0]
```

Comment:
It seems there is an API from Megatron-LM for retrieving the model-parallel rank - i.e., ``megatron.core.parallel_state.get_model_parallel_src_rank``.

Reply:
I've also checked the `megatron.core.parallel_state.get_model_parallel_src_rank` from the main branch.
Since we are using core_r.0.9.0 and there isn't `get_model_parallel_src_rank`, I suggest  we should use `torch.distributed.get_process_group_ranks`

Reply:
I see, we should use ``torch.distributed.get_process_group_ranks`` for now, and then replace it to ``megatron.core.parallel_state.get_model_parallel_src_rank`` after the version upgrade of NeMo.